fixtures/varytec/giga-bar-hex-3.json

```diff
@@ -0,0 +1,124 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Giga Bar HEX 3",
+  "categories": ["Color Changer", "Strobe", "Effect"],
+  "meta": {
+    "authors": ["Erik Pade"],
+    "createDate": "2021-08-11",
+    "lastModifyDate": "2021-08-11"
+  },
+  "links": {
+    "manual": [
+      "https://images.static-thomann.de/pics//atg/atgdata/document/manual/472654_c_472654_472655_r2_de_online.pdf"
+    ],
+    "productPage": [
+      "https://www.thomann.de/de/varytec_giga_bar_hex_3.htm"
+    ],
+    "video": [
+      "https://www.thomann.de/de/thomanntv_video_varytec_giga_bar_hex_3_8849.html"
+    ]
+  },
+  "rdm": {
+    "modelId": 0
+  },
+  "physical": {
+    "dimensions": [360, 150, 64],
+    "weight": 0.4,
+    "power": 36,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    }
+  },
+  "availableChannels": {
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Amber": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Amber"
+      }
+    },
+    "UV": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "UV"
+      }
+    },
+    "Intensity": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Strobe": {
+      "capability": {
+        "type": "NoFunction"
+      }
+    },
+    "Reserved": {
```

Comment:
Reserved seems to be the wrong name as this contains multiple different (auto) effects.